### DISCUSSION TITLE
New enemy Idea "Cyborg duck"

### DISCUSSION THREAD

**Discussion by Daria13szamman** [Quit975/DuckyGame#49] [Ideas]
Let's add cyborg ducks with laser eyes. They should appear later in the game for the element of surprise. Add this as boss music https://youtu.be/zMUSTnjLmXQ  (This music was recommended by zielonyludek) . Just an idea :)

**Comment by Quit975** [comment]
And again, the enemies could be different depending on where the game takes place (https://github.com/Quit975/DuckyGame/discussions/55) - so maybe in some factory environment?

Later on, after we have a prototype, you will probably need to provide more details about suggested enemy - it's behavior, movement or attack patterns, etc etc
